### PR TITLE
Make toolbar icons configurable

### PR DESCRIPTION
**Pull request by kevinhinterlong**
Fixes alignment of theme dropdown and scroll lock/edit label

image
image

### CODE REVIEW

File: packages/perspective-viewer-datagrid/src/less/toolbar.less [hunk 1]
```diff
@@ -87,6 +87,8 @@
 
 .button:hover {
     opacity: 1;
+    display: flex;
+    align-items: center;
```
Comment (texodus) [inline packages/perspective-viewer-datagrid/src/less/toolbar.less:91]: What's the context for this?  The children of the elements this class is applied are the same height as the container so this should be inert - and indeed testing this locally against FF+Chrome+Safari and I see no difference?

**Review by texodus — COMMENTED** [review]
Thanks for the PR!

Can you please share some details on the intent behind the `flex` changes?  These are inert in the current DOM structure, and I don't see any difference in your screenshots or my testing to `1.3.5`, so I expect that you're doing something unexpected (or unintended) with your custom styling.

File: packages/perspective-viewer-datagrid/src/less/toolbar.less (hunk 1 above)
Comment (kevinhinterlong) [reply packages/perspective-viewer-datagrid/src/less/toolbar.less:91]: This was to fix the alignment of the text/selectors in these buttons. Visiting https://perspective.finos.org/ on Firefox I'm able to reproduce this, but I just tested Chrome and Safari and I don't see the issue there

theme selector alignment
scroll lock alignment

Comment (jkusa) [reply packages/perspective-viewer-datagrid/src/less/toolbar.less:91]: I am seeing the same as @kevinhinterlong on FF with MacOS.

Comment (texodus) [reply packages/perspective-viewer-datagrid/src/less/toolbar.less:91]: Ok, I do see this on FF OSX 78.9, thanks for the clarification

**Comment by texodus** [comment]
Thanks for the PR!  Looks good!

**Pull request merged**